tests/features/conftest.py

```diff
@@ -44,6 +44,17 @@ def ten_lines(tmpdir):
     return f
 
 
+@pytest.fixture
+def three_lines_with_indentation(tmpdir):
```

Comment:
if this is only used in that particular file put it in there

Reply:
Not sure I'm following you on this...
Where do you want me to put this ?

Reply:
in tests/features/comment_test